Comparison/src/cz/fidentis/comparison/procrustes/GPA.java

```diff
@@ -178,6 +178,10 @@ public void superimpose(){
         
         for(int i = 0; i < configs.size(); i++){
             ICPTransformation tran = configs.get(i).rotate(oldMean);  
+            
+            if(tran == null)

```

Comment:
Stava sa to v pripade, ze bolo najdenych menej ako 3 korespondujuce body medzi 2 konfiguraciami. Empty list by mozno bol lepsie riesenie, teraz som si spomenula, ze sa to pouziva na jednom mieste, kde som sa zabudla pozriet, opravim updatnem atd.